sample_information_validator.py

```diff
@@ -1,20 +1,22 @@
-#!/usr/bin/env python
-"""
-Validates the user submitted sample information sheet
-so that no errors are propagated downstream.
-"""
+# load libraries
```

Comment:
I'm not very good with Python, but I do think it's a good idea to keep that at the beginning of the script:
#!/usr/bin/env python